Sources/TuistKit/Commands/EncryptCommand.swift

```diff
@@ -0,0 +1,48 @@
+import Basic
+import Foundation
+import SPMUtility
+import TuistCore
+import TuistSigning
+import TuistSupport
+
+class EncryptCommand: NSObject, Command {
+    // MARK: - Attributes
+
+    static let command = "encrypt"
+    static let overview = "Encrypts all files in Tuist/Signing directory."
+    private let pathArgument: OptionArgument<String>
+
+    private let signingCipher: SigningCiphering
+
+    // MARK: - Init
+
+    public required convenience init(parser: ArgumentParser) {
+        self.init(parser: parser, signingCipher: SigningCipher())
+    }
+
+    init(parser: ArgumentParser,
+         signingCipher: SigningCiphering) {
+        let subParser = parser.add(subparser: EncryptCommand.command, overview: EncryptCommand.overview)
+        pathArgument = subParser.add(option: "--path",
+                                     shortName: "-p",
+                                     kind: String.self,
+                                     usage: "The path to the folder where the template will be generated (Default: Current directory).",
```

Comment:
This usage text is incorrect, how about:

> The path to the folder containing the certificates you would like to encrypt

Reply:
Done 👍